portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
```

Comment:
Oh cool I just noticed the img tags later follow this pattern already, so I think only this meta tag missed the /> treatment

Reply:
I never noticed that I used different styles, so I checked the style guide and ultimately decided to use > since some of the code from the original repo used > as well as the style not specifying a preference. I'll be sure to be consistent in the future with this.